pkg/codegen/testing/test/testdata/external-resource-schema/nodejs/argFunction.ts

```diff
@@ -23,10 +22,10 @@ export interface ArgFunctionResult {
     readonly age?: number;
 }
 
+
 export function argFunctionOutput(args?: ArgFunctionOutputArgs, opts?: pulumi.InvokeOptions): pulumi.Output<ArgFunctionResult> {
-    return pulumi.output(args).apply(a => argFunction(a, opts))
+    return pulumi.output(args).apply((a: any) => argFunction(a, opts))
```

Comment:
It would be really helpful if a change this big didn't change code for unchanged schemas! Could you apply this change to pu/pu and then rebase to cut down the number of files necessary to review here.

Reply:
Yeah I've opened #11511 for this